plugins/inlinetoolbar/skins/kama/inlinetoolbar.css

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+For licensing, see LICENSE.md or http://ckeditor.com/license
+*/
+
+.cke_inlinetoolbar
```

Comment:
I don't like the fact of re-implementing such a generic CSS again in the inline toolbar plugin.

Instead the balloon panel created by this plugin should **add** `cke_inlinetoolbar` class to that outer most element.

Then you'd need to adjust selector from `.cke_inlinetoolbar` to `cke_balloon.cke_inlinetoolbar (...)`.

The main goal is to avoid code redundancy.